src/khoj/database/models/__init__.py

```diff
@@ -84,23 +85,33 @@ class ModelType(models.TextChoices):
 
 
 class Agent(BaseModel):
-    creator = models.ForeignKey(
-        KhojUser, on_delete=models.CASCADE, default=None, null=True, blank=True
-    )  # Creator will only be null when the agents are managed by admin
+    creator = models.ForeignKey(KhojUser, on_delete=models.CASCADE, default=None, null=True, blank=True)
     name = models.CharField(max_length=200)
     personality = models.TextField()
     avatar = models.URLField(max_length=400, default=None, null=True, blank=True)
     tools = models.JSONField(default=list)  # List of tools the agent has access to, like online search or notes search
     public = models.BooleanField(default=False)
     managed_by_admin = models.BooleanField(default=False)
     chat_model = models.ForeignKey(ChatModelOptions, on_delete=models.CASCADE)
+    slug = models.CharField(max_length=200, default=None, null=True, blank=True)
```

Comment:
In what scenarios is the agent slug expected to be unset?

Reply:
Hmm, shouldn't be. Good point, I'll update the model.